<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRangeSlider" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>431</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="setVtvar_setting">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>XXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Vt_label_set">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>199</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>Vt (ml)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="setVtvar_setting_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>XXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="setVtvar_setting_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>99</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>XXXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Vt_label_set_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>-1</y>
     <width>199</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>PIP (cmH2O)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRangeSlider" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>69</y>
     <width>51</width>
     <height>431</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="setVtvar_setting_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>439</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>XXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="setVtvar_setting_5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>XXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRangeSlider" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>51</width>
     <height>431</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="setVtvar_setting_6">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>XXXXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Vt_label_set_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>199</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(3, 43, 91);</string>
   </property>
   <property name="text">
    <string>Vt (ml)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRangeSlider</class>
   <extends>QProgressBar</extends>
   <header>qrangeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
